### PR TITLE
fan-page 4 

### PR DESCRIPTION
**Pull request by brianandreasen1983**
fan-page 4 re-uploaded the images I had used as in fan-page 3 they were not correct for one key image in the header.

### CODE REVIEW

**Comment by brianandreasen1983** [comment]
last minute revisions based on feedback from fan-page 3.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+body {
+    font-family: helvetica, tahoma, monospace;
+    background-color: #000000;
+    background-image: linear-gradient(to bottom, #000000, #FF0000, #000000);
```
Comment (mbMosman) [inline style.css]: gradient? :heavy_check_mark:

File: index-alt.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+<head>
+    <title>Reflex an Arena FPS game</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style-alt.css">
+</head>
+
+<div id="wrapper">
+
+    <body>
```
Comment (mbMosman) [inline index-alt.html]: Need to start the body before the above div.  This is a validation error.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    background-repeat: no-repeat;
+    background-position: center;
+    align-content: center;
+    background-size: contain;
+    background-clip: content-box;
+}
+
+h2,
+h3 {
+    text-transform: uppercase;
```
Comment (mbMosman) [inline style.css]: uppercase headings? :heavy_check_mark:

File: style.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+nav a {
+    text-decoration: underline;
+    color: #FFFFFF;
+}
+
+p {
+    color: #fff500;
+    text-indent: 2em;
```
Comment (mbMosman) [inline style.css]: text indent on paragraphs :heavy_check_mark:

File: style.css [hunk 5]
```diff
@@ -0,0 +1,88 @@
+body {
+    font-family: helvetica, tahoma, monospace;
+    background-color: #000000;
+    background-image: linear-gradient(to bottom, #000000, #FF0000, #000000);
+    color: #ffffff;
+}
+
+h1 {
```
Comment (mbMosman) [inline style.css]: standout h1? It is split between the 2 styles.  :heavy_check_mark:

File: style-alt.css [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    background-color: #ebbb1a;
+    background-image: url(images/horizontal-white.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    background-clip: content-box;
+    border-top-left-radius: 60px;
+    border-top-right-radius: 60px;
+    border-bottom-left-radius: 60px;
+    border-bottom-right-radius: 60px;
```
Comment (mbMosman) [inline style-alt.css]: rounded corners? :heavy_check_mark: 

You can combine this into one line: `border-radius: 60px;`

File: style-alt.css [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+a {
+    color: olive;
+}
+
+main {
+    background-image: url(images/screenshot00001.jpg);
+    background-repeat: no-repeat;
+    background-position: left top;
+    width: 80%;
+    margin-left: 100px;
```
Comment (mbMosman) [inline style-alt.css]: to center this, you would need to do what you've done in the wrapper below.  Give it a width and use auto margins.

File: style-alt.css [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+    opacity: .5;
+}
+
+ul {
+    list-style: circle;
+    color: #696d00;
+}
+
+footer {
+    text-align: center;
```
Comment (mbMosman) [inline style-alt.css]: centered text? :heavy_check_mark:

File: style-alt.css [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+    width: 80%;
+    margin-left: 100px;
+}
+
+p {
+    border: 5px solid #FF0000;
+    border-bottom-right-radius: 110px;
+    border-bottom-left-radius: 110px;
+    border-top-right-radius: 110px;
+    border-top-left-radius: 110px;
```
Comment (mbMosman) [inline style-alt.css]: rounded corners on para's? :heavy_check_mark:

File: style-alt.css [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+    text-indent: 1em;
+    color: #c6d32d;
+    opacity: .5;
```
Comment (mbMosman) [inline style-alt.css]: You'd need a background color for the paragraphs to make the text a bit more readable and have the opacity work as expected.

**Pull request closed without merging**